### DISCUSSION TITLE
If it is possible to upload posts with multiple photos ?

### DISCUSSION THREAD

**Discussion by daniilberkutdev** [diezo/Ensta#37] [Q&A] [answered]
I tried to create a list of photo's path, tuple of photos.
If this is not possible on the current version, then please add this feature in the future. All the best to the project, thanks.

**Comment by diezo** [comment]
although it's not possible on the current version, i'll add the implementation very soon!

btw, don't forget to star this repository so that other developers can know about ensta as well.

thank you for the suggestion!

**Reply by daniilberkutdev** [reply]
Already starred ;)
Thanks for feedback

**Reply by diezo** [reply]
hey! i've added the feature. please check the discussion thread.

**Comment by diezo** [comment, marked as answer]
hey @daniilberkut

i've added the feature to post multiple photos in one single post. here's the code:

```py
from ensta import Host

host = Host(username, password)

upload1 = host.get_upload_id("1.jpeg")
upload2 = host.get_upload_id("2.jpeg")
upload3 = host.get_upload_id("3.jpeg")

host.upload_photos([upload1, upload2, upload3], caption="Summer vacation!")
```

are you satisfied with the implementation?
